home_assistant_streamdeck_yaml.py

```diff
@@ -1863,14 +2424,41 @@ async def run(
     """Main entry point for the Stream Deck integration."""
     deck = get_deck()
     async with setup_ws(host, token, protocol) as websocket:
-        complete_state = await get_states(websocket)
-        update_all_key_images(deck, config, complete_state)
-        deck.set_key_callback_async(
-            _on_press_callback(websocket, complete_state, config),
-        )
-        deck.set_brightness(config.brightness)
-        await subscribe_state_changes(websocket)
-        await handle_changes(websocket, complete_state, deck, config)
+        try:
+            complete_state = await get_states(websocket)
+
+            if config.brightness_entity_id is not None:
+                brightness = int(
+                    float(complete_state[config.brightness_entity_id]["state"]),
+                )
+                if brightness >= 0 and brightness <= 100:
```

Comment:
I saw this brightness check before, please move it to a separate function `check_brightness_boundaries` or consider using constants for the boundaries